sql/core/src/test/scala/org/apache/spark/sql/UDFSuite.scala

```diff
@@ -247,4 +248,21 @@ class UDFSuite extends QueryTest with SharedSQLContext {
       sql("SELECT tmp.t.* FROM (SELECT testDataFunc(a, b) AS t from testData2) tmp").toDF(),
       testData2)
   }
+
+  test("Column aliases are ignored in callUDF while using struct()") {
```

Comment:
Yea, thanks! As I said, this is an old PR, maybe there is already other change fixing this issue. Let me check it today. If so, I will close this.